.github/workflows/lint_pr_title.yml

```diff
@@ -0,0 +1,22 @@
+# DESCRIPTION:
+#  A workflow used to verify if PR titles matches conventional commits strategy.
+# END
+
+name: Lint PR Title
+run-name: "Lint PR - (#${{ github.event.number }}) ${{ github.event.pull_request.title }}"
+
+permissions:
+  pull-requests: read
+
+on:
+  pull_request_target:
```

Comment:
Shall we target it only to be run against the main branch?

Reply:
well, we could, why not